packages/oidc_core/lib/src/managers/user_manager_base.dart

```diff
@@ -731,10 +731,12 @@ abstract class OidcUserManagerBase {
   ///
   /// An [OidcException] will be thrown if the server returns an error.
   Future<OidcUser?> refreshToken({
+    OidcToken? overrideToken,
```

Comment:
I think the `overrideToken` is unnecessary here, as you said, shared code should go into a `doRefreshToken` method that's marked `@protected`